newrelic/validation.go

```diff
@@ -6,6 +6,24 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+func validateViolationCloseTimer() schema.SchemaValidateFunc {
+	return func(i interface{}, k string) (warnings []string, errors []error) {
+		val, ok := i.(int)
+		if !ok {
+			errors = append(errors, fmt.Errorf("expected type of %s to be int", k))
+			return
+		}
+		switch val {
+		case 1, 2, 4, 8, 12, 24, 48, 72:
+		case 0:
+			warnings = append(warnings, "0 is no longer a valid value. Using the default value of 24")
```

Comment:
Where do we set the default 24 value? I'm not seeing it in the changes.

Reply:
The default value is enforced in the API itself. We just want to give a warning here so we don't break a customer's Terraform project.